lib/Sema/CSApply.cpp

```diff
@@ -114,7 +114,7 @@ Type Solution::computeSubstitutions(
                          getConstraintSystem().DC,
                          (ConformanceCheckFlags::InExpression|
                           ConformanceCheckFlags::Used));
-    (void)isOpenedAnyObject;
+    (void)&isOpenedAnyObject;
     assert((conformance ||
```

Comment:
Out of curiosity, what diagnostic are you getting from VC++ that requires taking the address here?